Update/standard app to template app (#862)
MapsPeople/docs: src/product/index.md

The MapsIndoors SDKs are the engines that ensure your users can view, navigate, and receive important information about the space around them. Common use cases include getting directions from an outdoor location to your indoor locations, searching your indoor locations, showing the relevant information for a specific location, exposing dynamic data for a location, and more. An exhaustive list of features within the SDKs can be found in the reference documentation for each platform.

## MapsIndoors Templates

> The MapsIndoors Template is both a full app and plug-and-play code for you to integrate basic usage of MapsIndoors, containing search and directions functionalities, in your existing app. If you just want to get started with a simple solution with no customisation, this should fulfil your needs. Most clients choose to only use the Template for testing or demo purposes, and use the SDKs to create a completely customized experience for their users.

Reviewer: I'm quite concerned about using the term "full app" here, as I think it could lead to some confusion on its exact purpose. I'm aware that it can stand alone, or be a "page" in an existing app, but we probably need more words on the fact that it's a starting point, not a destination.

Author: The first part of this paragraph is taken directly from where we first introduced the Template app, as an addition to the Getting Started guide. The wording has been through several rounds of review and approval, so if it's to be changed here, I think another larger discussion needs to be had about how we talk about and refer to the Template app in general 🙂

![WebApp](/assets/product/webApp.png)

> For web development, instead of the MapsIndoors Template, a MapsIndoors "Standard App" with similar functionality also includes a plug-and-play Kiosk web app. This browser-based app can run on an information stand and help one-time users of a facility find their way without needing to install an app.

Reviewer: ```suggestion
For web development, instead of the MapsIndoors Template, a MapsIndoors "Showcase App" with similar functionality, as well as a plug-and-play Kiosk web app exists.
```

Let's write less about the Kiosk 🙂
